index.html

```diff
@@ -720,12 +901,12 @@ <h2 class="subscribe-block__title title">
 
     <footer>
       <!-- Footer section -->
-      <section class="footer-section">
+      <section class="footer">
         <!-- Nav container -->
-        <div class="container footer-section__nav-container">
+        <div class="container footer__container">
           <!-- Footer dots -->
           <img
-            class="footer-dots-left"
+            class="footer-dots_left"
```

Comment:
```suggestion
            class="footer__dots_left"
```
As we use absolute positioning, `img` with this class can't be used outside the block, so `footer__footer-dots_left` or `footer__dots_left` will be more correct